ivy/functional/frontends/tensorflow/math.py

```diff
@@ -296,4 +296,8 @@ def zero_fraction(value, name="zero_fraction"):
     return ivy.divide(count_zero, ivy.add(count_zero, count_nonzero))
 
 
+def argmin(input, axis, output_type, name=None):
```

Comment:
Please make sure that the default values are the same as in the original frontend function https://www.tensorflow.org/api_docs/python/tf/math/argmin